app/scripts/administration/controllers/administration.controller.js

```diff
@@ -0,0 +1,20 @@
+/*jshint -W003, -W098, -W033 */
+(function () {
+  'use strict';
+
+  /**
+   * @ngdoc function
+   * @name ngAmrsApp.controller:MainCtrl
+   * @description
+   * # MainCtrl
+   * Controller of the ngAmrsApp
+   */
+  angular
+    .module('app.administration')
+    .controller('AdministrationCtrl', AdministrationCtrl);
+  AdministrationCtrl.$nject = ['$rootScope', '$scope'];
+
+  function AdministrationCtrl($rootScope, $scope) {
```

Comment:
Followed patient dashboard design where we have a main controller that serves several other directives